src/interpreter.cc

```diff
@@ -116,25 +116,39 @@ namespace rego
 
   Node Interpreter::get_errors(const Node& node) const
   {
+    if (!has_err())
+      return {};
+    reset_err();
+
+    Node errorseq = NodeDef::create(ErrorSeq);
```

Comment:
If an error object was used, then adding errors to a `Nodes` vector could be something it does when the `err()` method is called.